tests/setup/sg_init.sh

```diff
@@ -73,6 +73,7 @@ if [ "${IS_ES_SECURED:-}" = true ] ; then
     cp ${TMP_DIR}/search-guard-ssl/example-pki-scripts/transport/node-0-keystore.jks ${ES_HOME}/plugins/search-guard-2/sgconfig/transport-node-0-keystore.jks
     cd ${ACTUAL_DIR}
 
+    cp ${ES_CONF}/sg_*.yml ${ES_HOME}/plugins/search-guard-2/sgconfig
```

Comment:
They are slightly different.  They give us the ability to test with a sg config more similar to openshift, if desired.